MIVisionX-setup.py

```diff
@@ -427,8 +427,11 @@
             os.system('sudo -v')
             if os.path.exists(ROCM_PATH+'/rpp'):
                 os.system('sudo rm -rf '+ROCM_PATH+'/rpp')
+            rppCMakeArg = ''
+            if backend == 'HIP':
+                    rppCMakeArg = ' -DCOMPILE=HIPRTC'
```

Comment:
@kiritigowda please remove this line as it is no longer needed to build RPP for the HIP backend. 

To build RPP for the HIP backend in master or the 0.8 branches of the RPP, we just need to use cmake -DBACKEND=HIP ..